.github/workflows/code-examples.yml

```diff
@@ -51,6 +51,7 @@ jobs:
         id: check-modified-paths
         if: github.event_name != 'workflow_dispatch'
         with:
+          base: 'develop'
```

Comment:
Was this added on purpose or just for debugging? It looks a bit excessive, because (if I understand) correctly, it will report all changes on a branch, not only all changes in the current commit.

Reply:
Explained in https://github.com/handsontable/handsontable/pull/9184#issuecomment-1028771754